OCCommunicationLib/OCCommunicationLib/OCCommunication.h

```diff
@@ -57,7 +57,8 @@ typedef enum {
     OCErrorPrivateLinkRedirectionFailed = 1200,
     OCErrorPrivateLinkFileNotExists = 1201,
     OCErrorPrivateLinkFileNotCachedOffline = 1202,
-    OCErrorPrivateLinkErrorCachingFile = 1203
+    OCErrorPrivateLinkErrorCachingFile = 1203,
+    OCErrorPrivateLinkErrorCachingFile4 = 1204
```

Comment:
I think we changed this error for debug purposes in an old branch and you left it here. Unless it's still used somewhere (then we should change the name) we should delete it.